DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml

```diff
@@ -32,8 +32,32 @@
   <tr role="row" class="nhsuk-table__row first-row">
     <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
       <span class="nhsuk-table-responsive__heading">@competency.Vocabulary </span>@competency.Name
-      @if (competency.AlwaysShowDescription) {
-    <p class="nhsuk-u-margin-top-2">@Html.Raw(competency.Description)</p>
+      @if (competency.Description != null && !competency.AlwaysShowDescription)
+      {
+        <details class="nhsuk-details">
```

Comment:
Can we use the NHSUK details implementation (https://service-manual.nhs.uk/design-system/components/details) here?